Common/Utils/include/CommonUtils/DebugStreamer.h

```diff
@@ -141,7 +142,7 @@ class DebugStreamer
   std::string getUniqueTreeName(const char* tree, const size_t id = getCPUID()) const;
 
   /// set directly the debug level
-  static void setStreamFlags(const StreamFlags streamFlags) { o2::conf::ConfigurableParam::setValue("DebugStreamerParam", "streamLevel", static_cast<int>(streamFlags)); }
+  static void setStreamFlags(const StreamFlags streamFlags) { o2::conf::ConfigurableParam::setValue<int>("DebugStreamerParam", "streamLevel", static_cast<int>(streamFlags)); }
```

Comment:
What this is needed, for me it works as ` o2::conf::ConfigurableParam::setValue("DebugStreamerParam", "streamLevel",..`  also, as well as `o2::utils::ParameterDebugStreamer::Instance().setValue("streamLevel","7");`

Exposing boost stuff in the header may lead to some problems.